command/snapshot/inspect/snapshot_inspect.go

```diff
@@ -56,27 +65,181 @@ func (c *cmd) Run(args []string) int {
 	}
 	defer f.Close()
 
-	meta, err := snapshot.Verify(f)
+	readFile, meta, err := snapshot.Read(nil, f)
```

Comment:
The first arg is a `Logger`, right? I think this is going to panic if any errors are encountered because logger is nil. I guess we don't have a logger yet, but I think a default logger (`hclog.New(nil`) would be fine for now.